src/Servers/Kestrel/Core/src/Internal/Infrastructure/PipeWriterHelpers/TimingPipeFlusher.cs

```diff
@@ -92,7 +92,7 @@ private async ValueTask<FlushResult> TimeFlushAsyncAwaited(ValueTask<FlushResult
         }
         catch (OperationCanceledException ex) when (outputAborter is object)
         {
-            outputAborter.Abort(new ConnectionAbortedException(CoreStrings.ConnectionOrStreamAbortedByCancellationToken, ex));
+            outputAborter.Abort(new ConnectionAbortedException(CoreStrings.ConnectionOrStreamAbortedByCancellationToken, ex), ConnectionEndReason.AbortedByApp);
```

Comment:
```suggestion
            outputAborter.Abort(new ConnectionAbortedException(CoreStrings.ConnectionOrStreamAbortedByCancellationToken, ex), ConnectionEndReason.CanceledFlush);
```

I think "canceled_flush" is worth separating from "aborted_by_app". If we do this, "aborted_by_app" should definitively indicate that `HttpContext.Abort()` was called on an HTTP/1.1 connection which seems useful for finding the root cause of aborted connections.

Reply:
Since it is to do with writing, what about `canceled_write`? That makes it a bit more immediately obvious to more people what the general problem area is. Flushing the body is usually handled by frameworks.

Reply:
Good points. I now like `canceled_write` better than `canceled_flush` too.